vite.config.ts

```diff
@@ -21,19 +21,21 @@ const createBanner = () => {
 export default defineConfig({
   plugins: [
     react(),
+    eslint(),
     dts({ staticImport: true, outputDir: './dist/.temp' }),
-    externalizeDeps(),
   ],
   build: {
     lib: {
       entry: resolve(__dirname, 'src/index.ts'),
       formats: ['es', 'cjs'],
     },
     rollupOptions: {
+      external: ["react", "react-dom", "baseui", "styletron-react"],
```

Comment:
build files is now content react+base ui in our code. It shouldn't happen before. I propose ytou to return externalize deps and config like before.